library/fiber/raise.rb

```diff
@@ -0,0 +1,91 @@
+require_relative '../../spec_helper'
```

Comment:
Fiber#raise is actually a core method (it does not need `require 'fiber'` unlike e.g. Fiber#transfer), so it should be in `core/fiber/raise_spec.rb`